backends/kiibohd.py

```diff
@@ -244,11 +251,20 @@ def process( self, capabilities, macros, variables, gitRev, gitChanges ):
 					# TODO Add support for Analog keys
 					# TODO Add support for LED states
 					self.fill_dict['TriggerMacros'] += "0x00, 0x01, 0x{0:02X}, ".format( triggerItem )
+					triggers_count += 1
 
 			# Add list ending 0 and end of list
 			self.fill_dict['TriggerMacros'] += "0 };\n"
 		self.fill_dict['TriggerMacros'] = self.fill_dict['TriggerMacros'][ :-1 ] # Remove last newline
 
+		print( "Triggers count:", triggers_count )
+
+		# check for too small stateWordSize
+		if stateWordSize == "8" and (triggers_count > 255 or results_count > 255):
+			print ("{0} Over 255 trigger or result macros, changing stateWordSize to 16.".format( WARNING ) )
```

Comment:
The warning should include whatever the stateWordSize was originally set to.
Perhaps include why we are changing it. (i.e. Print out a table with the various counting results, since it's a warning, it's ok to print a larger message).